icu4j/main/classes/core/src/com/ibm/icu/text/BreakIteratorFactory.java

```diff
@@ -131,15 +131,23 @@ private static BreakIterator createBreakInstance(ULocale locale, int kind) {
         ByteBuffer bytes = null;
         String typeKeyExt = null;
         if (kind == BreakIterator.KIND_LINE) {
-            String lbKeyValue = locale.getKeywordValue("lb");
-            if ( lbKeyValue != null && (lbKeyValue.equals("strict") || lbKeyValue.equals("normal") || lbKeyValue.equals("loose")) ) {
-                typeKeyExt = "_" + lbKeyValue;
+            String keyValue = locale.getKeywordValue("lb");
+            if ( keyValue != null && (keyValue.equals("strict") || keyValue.equals("normal") || keyValue.equals("loose")) ) {
+                typeKeyExt = "_" + keyValue;
+            }
+            String language = locale.getLanguage();
+            if (language != null && language.equals("ja")) {
+                keyValue = locale.getKeywordValue("lw");
+                if (keyValue != null && keyValue.equals("phrase")) {
+                    typeKeyExt = "_" + keyValue;
```

Comment:
It looks like the lw value *replaces* the lb value. Does this work? Shouldn't it append?
```suggestion
                    typeKeyExt += "_" + keyValue;
```